mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -17,13 +17,12 @@ impl BridgeController {
         Self::default()
     }
 
-    pub async fn start(&mut self) -> Result<()> {
+    pub async fn start(&mut self, system_address: String, device_id: &str) -> Result<()> {
         info!("starting bridge");
-
         let settings = Settings::new()?;
         if let Some(upstream) = settings.upstream() {
```

Comment:
If there are no upstream settings then we log:

> "No nested bridge found"

I think we should say something like:

> "No upstream settings detected. Not starting bridge.